pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -51,9 +52,14 @@ var _ = Describe("Node-labeller ", func() {
 		config, _, _ := testutils.NewFakeClusterConfigUsingKV(kubevirt)
 		recorder := record.NewFakeRecorder(100)
 		recorder.IncludeObject = true
+		cpuCounter := &libvirtxml.CapsHostCPUCounter{
+			Name:      "tsc",
+			Frequency: 4008012000,
+			Scaling:   "no",
+		}
```

Comment:
It could be nice to add a test for it.

Reply:
It's already being covered by `cpu_plugin_test.go`, theoretically the cpuCounter value could be nil for this test and it wouldn't matter. With that said, most of the existing tests for nodelabeller could be improved upon, but I think that would be better saved for another PR.

Reply:
After talking about it offline we've agreed that a test should be added as part of the nodelabeller logic.